src/app/components/EnterPage/enter.scss

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css?family=Mulish');
+
+:local {
+  .enter {
+    position: fixed;
+    width: 426px;
+    height: 238px;
+    left: calc(747/1920*100%);
+    top: 96px;
+
+    padding: 0px;
+    background: #FFFFFF;
+
+    box-shadow: 0px 0px 12px rgba(188, 188, 188, 0.25);
+
+    a {
+      text-decoration: none;
+      color:#1C83C6;
+    }
+
+    h5 {
+      position: absolute;
+      left: 36px;
+      top: 24px;
```

Comment:
Avoid using `position: absolute` for simple components.

Reason: The design file is only valid at the written time. In the future, there might be requests like adding one line at the top. For such request, when overuse `position: absolute`, all the elements on the component will require manual changes.

Suggestion: use `display: static` (default), or `display: flex` when possible, use `margin`, `padding` instead of `top`, `left`, `right`, `bottom`.